src/org/thoughtcrime/securesms/connect/DcHelper.java

```diff
@@ -438,4 +447,20 @@ public static String getConnectivitySummary(Context context, int connectedString
           return context.getString(R.string.connectivity_not_connected);
       }
   }
+
+  public static void showVerificationBrokenDialog(Context context, String name) {
+    new AlertDialog.Builder(context)
+            .setMessage(context.getString(R.string.chat_protection_broken_explanation, name))
+            .setNeutralButton(R.string.learn_more, (d, w) -> IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/684/en/help#verificationbroken"))
+            .setNegativeButton(R.string.qrscan_title, (d, w) -> context.startActivity(new Intent(context, QrActivity.class)))
+            .setPositiveButton(R.string.ok, null)
+            .setCancelable(true)
+            .show();
+  }
+
+  public static void showProtectionEnabledDialog(Context context) {
+    IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/684/en/help#whatdoesverifiedmean");
```

Comment:
i think, we should show an `AlertDialog` as for `showVerificationBrokenDialog()`, just to have the same and not unexpected behaviour. (maybe i argued the other way round at some point, but using it now, i think, this is better:)

but that can also go to another pr.